### ISSUE TITLE
Multiple in operators

### ISSUE THREAD

**Issue by phuongpt** [linq2js/linq2fire#2]
```
.where({
 text: ['text 1', 'text 2'],
 word: ['word 1', 'word 2'],
})
```
@linq2js looks like this case is missing, it takes the last one
BTW your library is so great! Thanks!

**Comment by linq2js** [comment]
@phuongpt thank for your feedback, let me take a look at this

**Comment by linq2js** [comment]
@phuongpt 
Could you provide your test cases, your criteria describes:
(text = 'text 1' or 'text 2') and (word = 'word 1' or word = 'word 2') , is this right ?

**Comment by phuongpt** [comment]
Yes @linq2js. That should be contain in operator.

**Comment by linq2js** [comment]
Fixed in linq2fire@1.0.6, so close this issue
